src/lib/party.ts

```diff
@@ -516,8 +575,105 @@ class Party {
     return new (Party as any)(body, authorization, party.isSelf);
   }
 
+  token(tokenId?: Field) {
+    let thisParty = this;
+    let customToken = new Token({
+      tokenOwner: thisParty.body.publicKey,
+      parentTokenId: tokenId,
+    });
+    return {
+      id: customToken.id,
+      parentTokenId: customToken.parentTokenId,
+      tokenOwner: customToken.tokenOwner,
+
+      mint({ address, amount }: MintOrBurnParams) {
+        let receiverParty = Party.createUnsigned(address, {
+          caller: this.id,
+          tokenId: this.id,
+          callDepth: 1,
+          useFullCommitment: Bool(true),
```

Comment:
this shouldn't set `useFullCommitment`, a receiver doesn't need replay protection

Reply:
Nice catch, fixed!